webapp/channels/src/packages/mattermost-redux/src/reducers/websocket.ts

```diff
@@ -33,10 +32,6 @@ export default function reducer(state = getInitialState(), action: AnyAction) {
         return getInitialState();
     }
 
-    if (action.type === UserTypes.LOGOUT_SUCCESS) {
```

Comment:
Why is this removed?

Reply:
There's actually an exact copy of this directly above it. I think it must've been introduced by a bad merge at some point, but we never noticed it. I only came across it because TS starts complaining once the changes in #29486 are made since it knows that `action.type` can't have that value at this point